README.md

```diff
@@ -14,8 +14,14 @@ Prerequesites:
 
 * A valid Equinix Metal organization ID, API token, and/or project ID.
 * HashiCorp Terraform installed. Please see [tfenv](https://github.com/tfutils/tfenv) util.
+* Install [pre-commit](https://pre-commit.com/#install) with its prerequesites: [python](https://docs.python.org/3/using/index.html) and [pip](https://pip.pypa.io/en/stable/installation/).
```

Comment:
- https://docs.python.org/3/using/unix.html#getting-and-installing-the-latest-version-of-python instead of the index page for python
- prerequesites -> prerequisites

Reply:
The two added steps should also be optional. GH CI will identify problems if the user doesn't catch them on pre-commit (all but the shfmt and markdown linting?). If we are going to enforce style on shell scripts and markdown docs that should be done in CI. (the precommit helper would then be an optional way for developers to get ahead of the CI)

Reply:
Thank you! Errors fixed in [26f7c22 ](https://github.com/equinix-labs/terraform-equinix-labs/commit/26f7c22d7c375ec399590ce623f76d6224375376)